src/components/Person/Person.jsx

```diff
@@ -1 +1,20 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      My name is {person.name}
+    </h2>
+
+    {person.age >= 0 && (
+      <p className="Person__age">I am {person.age}</p>
+    )}
+
+    <p className="Person__partner">
+      {person.isMarried ?
+        `${person.partner.name} is my
+          ${person.partner.sex === 'f' ?
+            "wife":
+            "husband"}` :
+        "I am not married"}
```

Comment:
The `person.partner.name` and `person.partner.sex` properties are accessed here, but the `person` object does not have a `partner` property in the current structure. You should access `person.partnerName` and use a condition to determine the partner's sex based on the `person.sex` property.